### PR TITLE
Use MO as module

### PR DESCRIPTION
**Pull request by dkurt**
For https://github.com/openvinotoolkit/open_model_zoo/pull/2190

Just
```
python -m mo --input_model ...
```

### CODE REVIEW

**Comment by dkurt** [comment]
Confusing error message:
```
[ ERROR ] File /abc/def does not exist
[ ERROR ] Failed to convert tokens to dictionary: Wrong character "}" in position 2
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] Failed to convert tokens to dictionary: Wrong character "}" in position 5
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] Failed to convert tokens to dictionary: Wrong character "}" in position 5
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] Failed to convert tokens to dictionary: list index out of range
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] Failed to convert tokens to dictionary: Input/output braces mismatch.
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] Failed to convert tokens to dictionary: list index out of range
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] Failed to convert tokens to dictionary: Missing value in the list at position 10
[ ERROR ] Failed to generate dictionary representation of file.
[ ERROR ] File /non/existing/file does not exist
[ ERROR ] Failed to read file /tmp/tmpwk4rnpof: [Errno 13] Permission denied: '/tmp/tmpwk4rnpof'

FAIL [0.029s]: test_bom_file (mo.bom_test.TestBOMFile)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/openvino/openvino/model-optimizer/mo/bom_test.py", line 96, in test_bom_file
    self.assertTrue(not len(missing_files), '{} files missed in BOM'.format(len(missing_files)))
AssertionError: False is not true : 1 files missed in BOM
```

File: model-optimizer/mo/__main__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import sys
```
Comment (lazarevevgeny) [inline model-optimizer/mo/__main__.py:1]: Why you didn't add the legal header for this file? I think that we must do this if the file structure allows us to do this which I believe is true.

Comment (dkurt) [reply model-optimizer/mo/__main__.py:17]: Fair point

**Review by lazarevevgeny — APPROVED** [review]

File: model-optimizer/automation/package_BOM.txt [hunk 2]
```diff
@@ -769,6 +769,7 @@ install_prerequisites/protobuf-3.6.1-py3.6-win-amd64.egg
 install_prerequisites/protobuf-3.6.1-py3.7-win-amd64.egg
 mo.py
 mo/__init__.py
+mo/__main__.py
```
Comment (mryzhov) [inline model-optimizer/automation/package_BOM.txt:772]: Will it allow to configure the entry-points then?

Comment (dkurt) [reply model-optimizer/automation/package_BOM.txt:772]: I've tested only `python -m mo`

Comment (SDxKeeper) [reply model-optimizer/automation/package_BOM.txt:772]: Entrypoints is for mo pip package would be nice. Using them for OneZoo wheels.

Comment (dkurt) [reply model-optimizer/automation/package_BOM.txt:772]: Is that ok to make in a separate PR?

**Review by azhogov — APPROVED** [review]

File: model-optimizer/automation/package_BOM.txt (hunk 2 above)
Comment (mryzhov) [reply model-optimizer/automation/package_BOM.txt:772]: sure, do not mix those changes

**Review by mryzhov — APPROVED** [review]

**Pull request merged**